Vostok.ServiceDiscovery.Abstractions/ITag.cs

```diff
@@ -0,0 +1,23 @@
+using JetBrains.Annotations;
+
+namespace Vostok.ServiceDiscovery.Abstractions
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [PublicAPI]
+    public interface ITag

```

Comment:
Why does this interface even exist? What does it bring on top of the exceedingly simple `Tag` model?

Reply:
I really don't see a big need for this interface, Will delete it

Reply:
done